test/functional/common/phases.js

```diff
@@ -19,6 +19,8 @@ var Promise = require('bluebird');
 var apiHelpers = require('../../common/helpers/api');
 var waitFor = require('../../common/utils/wait_for');
 
+// TODO: Skip tests because WS batch transaction processing doesn't currently
+// send back errors or success statuses.
```

Comment:
I believe this confirmation phase is only for HTTP API. And every HTTP API endpoint is ending the status code.

Reply:
Whoops.